translations/messages.fr.xlf

```diff
@@ -0,0 +1,3061 @@
+        <source>This intersection is already occupied.</source>
+        <target>Cette intersection est déjà occupée.</target>
+      </trans-unit>
+      <trans-unit id="3878972107071324960" datatype="html">
+        <source>You cannot commit suicide.</source>
+        <target>Vous ne pouvez pas vous suicider.</target>
+      </trans-unit>
+      <trans-unit id="1472088308118018916" datatype="html">
+        <source>Preliminary information</source>
+        <target>Informations préalables</target>
+      </trans-unit>
+      <trans-unit id="352355973280326398" datatype="html">
+        <source>The game of Go is played on a board called a Goban, and the stones are placed on the intersections.
+        The traditional board is made of 19x19 intersections, but on this website we have the 13x13 board.
+        (For shorter parts, 9x9 and 5x5 boards exist, but are not yet available here).
+        For this tutorial, we will use a small 5x5 for pedagogical purposes.</source>
+        <target>Le jeu de Go se joue sur un plateau appelé Goban, et les pierres sont placées sur les intersections.
+        Le plateau traditionnel fait 19x19 intersections, mais le 13x13 est implémenté sur ce site.
+        (Pour des parties plus courtes, le 9x9 et 5x5 existent, mais ne sont pas encore disponibles).
+        Pour ce tutoriel, nous utiliserons un petit 5x5 à fin pédagogique.</target>
+      </trans-unit>
+      <trans-unit id="5188967317336954433" datatype="html">
+        <source>The goal of the game is to have the most points at the end of the game.
+        We call &quot;territories&quot; the intersections that are empty and isolated from the rest of the Goban by the stones of a single player.
+        Here, Dark has 9 territories on the left, and Light has 8 on the right.
+        The top part belongs to no one.
+        The score at the end of the game is the sum of the territories and the captures.</source>
+        <target>Le but du jeu est d&apos;avoir le plus de points en fin de partie.
+        On appelle territoires les intersections inoccupées et isolées du reste du Goban par les pierres d&apos;un seul joueur.
+        Ici, le joueur foncé a 9 territoires à gauche, le joueur clair en a 8 à droite.
+        La zone en haut au milieu n&apos;appartient à personne.
+        Le score en fin de partie correspondra à l&apos;addition des territoires et des captures.</target>
+      </trans-unit>
+      <trans-unit id="5239727864845885036" datatype="html">
+        <source>An isolated stone, like the one in the middle here, has 4 neighboring intersections (not 8, because we do not count diagonals).
+        If a neighboring intersection is unoccupied, it is called a freedom.
+        If Dark plays on the last freedom of the light stone, this stone is removed from the Goban (captured) and Dark earns one point.&lt;br/&gt;&lt;br/&gt;
+        There&apos;s only one freedom left to the light piece, capture it.</source>
```

Comment:
"it" est pas bizarrement ambigu, même si la pièce est mentionnées en dernier, ça fait un peu "capture la liberté", c'qui n'est pas correct ?
Ou c'est mon imagination ?

Reply:
tout à fait, car it réfère au dernier sujet qui est bien la liberté. Peut-être : The light piece only has one freedom left, capture it ?

Reply:
Ou bien "The light piece only has one liberty left, play there/play on it." ?